src/edu/ithaca/dragonlab/ckc/ConceptKnowledgeCalculator.java

```diff
@@ -42,38 +85,249 @@ public void clearAndCreateCohortData(String structureFilename, String resourceFi
+        cohortConceptGraphs = null;
+
+        //create the graph structure to be copied for each user
+
+        ConceptGraphRecord structureRecord = ConceptGraphRecord.buildFromJson(structureFileName);
+        List<LearningObjectLinkRecord> linkRecord = LearningObjectLinkRecord.buildListFromJson(resourceFile);
+        ConceptGraph graph = new ConceptGraph(structureRecord, linkRecord);
+
+        //old assessment file name
+        String oldAssessmentFile = assessmentFile;
+        CSVReader csvR = new CSVReader(oldAssessmentFile);
+        List<LearningObjectResponse> firstLOR = csvR.getManualGradedResponses();
+
+        //new assessment file name
+        CSVReader csvReader = new CSVReader(secondAssessmentFilename);
+        List<LearningObjectResponse> assessments = csvReader.getManualGradedResponses();
+
+
+        List<LearningObjectResponse> combinedAssessments = new ArrayList<>();
+        combinedAssessments.addAll(firstLOR);
+        combinedAssessments.addAll(assessments);
+
+
+        //create the average and individual graphs
+        cohortConceptGraphs = new CohortConceptGraphs(graph, combinedAssessments);
+
+        //to use in console
+        currentMode= Mode.COHORTGRAPH;
+        hasmultipleAssessmentFile=true;
+
+
+        //output the json representing the tree form of the graph
+        CohortConceptGraphsRecord toFile = cohortConceptGraphs.buildCohortConceptTreeRecord();
         String file = OUTPUT_PATH + "ckcCurrent.json";
         toFile.writeToJson(file);
     }
 
     @Override
     public String getCohortGraphsUrl() {
-        //TODO: need to find a way to offer a URL
-        return "To view graph, right-click index.html and choose \"open in Browser\" in ConceptKnowledgeCalculator/ckcvisualizer";
+
+            //TODO: need to find a way to offer a URL
+            return "To view graph, right-click index.html and choose \"open in Browser\" in ConceptKnowledgeCalculator/ckcvisualizer";
+
     }
 
+
+
+
     @Override
-    public String calcIndividualGraphSuggestions(String userId) {
-        if (cohortConceptGraphs != null) {
-            //TODO fix so that it returns new data type of two lists, rather than string, see interface
-            ConceptGraph userGraph = cohortConceptGraphs.getUserGraph(userId);
-            String suggestions = LearningObjectSuggester.buildSuggestionMap(userGraph).toString();
-            return suggestions;
+    public SuggestionResource calcIndividualGraphSuggestions(String userId) {
+        if (currentMode== Mode.COHORTGRAPH) {
+            if (cohortConceptGraphs != null) {
+                ConceptGraph userGraph = cohortConceptGraphs.getUserGraph(userId);
+                List<ConceptNode> concepts = LearningObjectSuggester.conceptsToWorkOn(userGraph);
+                return new SuggestionResource(userGraph, concepts);
+
+            } else {
+                return new SuggestionResource(null, null);
```

Comment:
default constructor to SuggestionResource with empty lists
-create test that calls default constructor